app/controllers/comments_controller.rb

```diff
@@ -4,13 +4,27 @@ class CommentsController < ApplicationController
   def show
   end
 
+  def create
+    status = CommentsCreation.new(comment_params['original_comments'], comment_params[:translated_comments], comment_params[:article_id]).rating
+
+    if status
+      render json: {
+        status: true
```

Comment:
usually status stands for status code, can you please change the name of this var